CODEOWNERS

```diff
@@ -134,7 +134,7 @@
 /boards/arm/mimxrt*/                      @mmahadevan108 @dleach02
 /boards/arm/mps2_an385/                   @fvincenzo
 /boards/arm/msp_exp432p401r_launchxl/     @Mani-Sadhasivam
-/boards/arm/npcx7m6fb_evb/                @MulinChao @ChiHuaL
+/boards/arm/npcx*/                        @MulinChao @ChiHuaL
```

Comment:
This is redundant actually, the maintainer file already covers this are, you could just drop it instead (or leave it as is, there's a PR in flight to remove entries covered by the maintainer file already). See the comment at the top.

Reply:
Thanks. Will drop this line in the PR later.

Reply:
Done.